events/setup.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import setuptools
+
+
+def read_requirements(file_):
```

Comment:
Same comment as for https://github.com/mozilla/code-review/pull/90, we probably don't need the additional complexity here